python/src/UniformOverMesh_doc.i.in

```diff
@@ -0,0 +1,77 @@
+%feature("docstring") OT::UniformOverMesh
+"UniformOverMesh distribution.
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Parameters
+----------
+mesh : :class:`~openturns.Mesh`
+    Mesh defining the domain of the distribution.
+
+Notes
+-----
+Its probability density function is defined as:
+
+.. math::
+
+    f_X(x) = \frac{1}{|\cD|}, \quad x \in \cD
+
+with :math:`\cD\subset\Rset^n` a mesh of dimension :math:`n`.
+
+Examples
+--------
+Create a distribution:
+
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> f = ot.SymbolicFunction(['x', 'y'], ['sin(x)*sin(y)'])
+>>> levelSet = ot.LevelSet(f, ot.Less(), 0.2)
+>>> box = ot.Interval([-5.0]*2, [5.0]*2)
+>>> mesh = ot.LevelSetMesher([50]*2).build(levelSet, box, False)
+>>> distribution = otexp.UniformOverMesh(mesh)
+
+Draw a sample:
+
+>>> sample = distribution.getSample(5)"
```

Comment:
There are two specific methods here: `getMesh` and `getIntegrationAlgorithm`. But the help page has more than 50 methods, because of class inheritance. I would suggest to add 1 or 2 example lines in the main example to show this, otherwise a user may not see that point.